projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -78,9 +78,6 @@ $styledInputs: 'input[type=text], input[type=password], input[type=number], inpu
     @include css-var(background-color, clr-stack-view-bg-color, $clr-stack-view-bg-color, $clr-use-custom-properties);
     word-wrap: break-word;
 
-    /* this fixes the overflow problem of children elements in a wrapper with a border-radius in Chrome */
```

Comment:
I was unable to reproduce this in any of our tested scenarios. I prefer to take this risk and fix a known and very prominent bug, even if it causes regression in a very theoretical corner case, with a Chrome version from many years ago.